app/views/shared/schools/mentors/_can_not_remove.html.erb

```diff
@@ -6,6 +6,22 @@
   <%= t(".title") %>
 </label>
 
-<p class="govuk-body">
   <%= simple_format(t(".description", mentor_name: mentor.full_name)) %>
-</p>
+
+  <ul class="govuk-list govuk-list--bullet">
+    <% mentor.placements.where(school:).decorate.each do |placement| %>
```

Comment:
Could we move this into a method?

Reply:
Ha! I said this, but as it's used only once I thought it was a bit much.